src/components/common/container.module.scss

```diff
@@ -18,4 +18,28 @@
   & .container-narrow {
     max-width: 42rem;
   }
+  &.text-heavy {
+    margin: 5em auto;
+    @media (min-width: $viewport-lg) {
+      font-size: 22px;
```

Comment:
How would you feel about converting this to ems, @schwartzadev? Or if you prefer, rems with a px fallback:

```
font-size: 22px;
font-size: 1.375rem;
```

Totally defer to you on what’s most feasible here! But a relative font size’ll be a bit more accessible.

Reply:
Great call!

I kept the px fallback and switched to rem in 83a50c606df97a02a50c56da914aca24f5ef2ce5